app/test_main.py

```diff
@@ -1 +1,35 @@
-# write your code here
+from unittest import mock
+
+import pytest
+
+from app.main import can_access_google_page
```

Comment:
also pay attention to import order. Make it according to PEP8

Reply:
IDK how it need to be? I re-read PEP8, its all about blank lines and order. I made changes after request by Serhii-Leonenko by "optimize imports" Now it's wrong too? I don't get it
<img width="425" alt="Screenshot 2023-02-27 at 21 02 11" src="https://user-images.githubusercontent.com/122725491/221657993-765e7eb2-dd3a-4628-897c-a7daaafbefac.png">
<img width="716" alt="Screenshot 2023-02-27 at 20 58 02" src="https://user-images.githubusercontent.com/122725491/221658054-b34367fc-5707-4854-9ab5-e211d532a4b4.png">
<img width="881" alt="Screenshot 2023-02-27 at 20 58 42" src="https://user-images.githubusercontent.com/122725491/221658076-2fddbba5-1823-4cad-ac7a-4a81eb66a85b.png">
<img width="597" alt="Screenshot 2023-02-27 at 20 58 47" src="https://user-images.githubusercontent.com/122725491/221658113-681ae938-e44b-46a0-b611-2b8c3162a771.png">